modules/shared-memory.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-ephemeral-storage.adoc
+
+[id="shared-memory_{context}"]
+= Shared Memory
+
+
+Shared memory is one of the fastest IPC (Inter-process communication) mechanisms in Linux. There are two types of shared memory objects: System V and POSIX.
```

Comment:
I might expand the acronym here, with the first reference.

Reply:
Addressed.